.github/workflows/windows.yml

```diff
@@ -32,10 +32,8 @@ jobs:
       run: |
         mkdir build
         cd build
-        cmake -D CMAKE_INSTALL_PREFIX=C:\kokkos-install \
-              -D Kokkos_ENABLE_THREADS=ON \
-              ..
-        cmake --build . --target install -- -m
+        cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_MSVC_RUNTIME_LIBRARY="MultiThreadedDebugDLL" -DCMAKE_INSTALL_PREFIX=C:\kokkos-install -DKokkos_ENABLE_THREADS=ON ..
+        cmake --build . --config Release --target install -- -m
```

Comment:
`Release` is the only option that worked for me here. Adding `CMAKE_MSVC_RUNTIME_LIBRARY` is necessary as otherwise the release and debug versions of the libraries will conflict when building ArborX in `Debug`.